airflow/timetables/interval.py

```diff
@@ -97,111 +96,95 @@ def next_dagrun_info(
         return DagRunInfo.interval(start=start, end=end)
 
 
-def _is_schedule_fixed(expression: str) -> bool:
-    """Figures out if the schedule has a fixed time (e.g. 3 AM every day).
-
-    :return: True if the schedule has a fixed time, False if not.
+class CronDataIntervalTimetable(_DataIntervalTimetable):
+    """
+    Timetable that schedules data intervals using one or more cron expressions.
 
-    Detection is done by "peeking" the next two cron trigger time; if the
-    two times have the same minute and hour value, the schedule is fixed,
-    and we *don't* need to perform the DST fix.
+    This corresponds to ``schedule_interval=<cron>``, where ``<cron>`` is either:
+    - A five-segment cron representation
+    - One of ``cron_presets``
+    - Or a collection containing values from the above
 
-    This assumes DST happens on whole minute changes (e.g. 12:59 -> 12:00).
+    Don't pass ``@once`` in here; use ``OnceTimetable`` instead.
     """
-    cron = croniter(expression)
-    next_a = cron.get_next(datetime.datetime)
-    next_b = cron.get_next(datetime.datetime)
-    return next_b.minute == next_a.minute and next_b.hour == next_a.hour
 
+    def __init__(
+        self, crons: Union[str, List[str], Set[str], Tuple[str, ...]], timezone: Union[str, Timezone]
+    ) -> None:
```

Comment:
Instead of normalizing here, I wonder if it’s easier to make this only accept `List[str]` and normalize in `DAG.__init__` instead.